update.sh

```diff
@@ -132,7 +132,7 @@ for version in "${versions[@]}"; do
 	echo "$version: $fullVersion"
 
 	for v in \
-		alpine{3.11,3.12} \
+		alpine{3.11,3.12,3.13} \
```

Comment:
I think we need to decide if we want to also drop Alpine 3.11 here (and keep in line with what we've done in Ruby and PHP and only support two at a time).

@yosifkit thoughts?

Reply:
Agreed, we can only support two versions of Alpine at a time. Added commit for that and updating `./generate-stackbrew-library.sh`